src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -51,20 +51,20 @@ private static Verification Verify(bool isStatic)
             return isStatic ? Verification.Skipped : VerifyOnMonoOrCoreClr;
         }
 
-        private static bool Execute(bool isStatic, bool haveImplementationInDerivedInterface = false)
+        private static bool Execute(bool isStatic, bool haveImplementationInDerivedInterface = false, bool hasImplementationOfStaticVirtualInDerivedType = false)
```

Comment:
>hasImplementationOfStaticVirtualInDerivedType [](http://example.com/codeflow?start=107&length=45)

It looks like the value passed at call sites doesn't match the name of the parameter. We are passing true if we have implementation of virtual in derived type, even for non-static scenario. Consider renaming the parameter to "hasImplementationOfVirtualInDerivedType" and adjusting condition of the ```if``` below to ```isStatic && hasImplementationOfVirtualInDerivedType```. Sorry I missed this during previous review pass

Reply:
Thanks. Fixed